eduVPN/HelperService.swift

```diff
@@ -151,10 +157,12 @@ class HelperService {
         connection?.exportedInterface = NSXPCInterface(with: ClientProtocol.self)
         connection?.exportedObject = client
         connection?.invalidationHandler = { () in
-            NSLog("connection invalidated!")
+            // Don't do this: handler(.failure(Error.connectionInvalidated))
+            // It causes the helper to fail to install (somehow)
         }
         connection?.interruptionHandler = { () in
-            NSLog("connection interrupted!")
+            // Don't do this: handler(.failure(Error.connectionInterrupted))
+            // It causes the helper to fail to install (somehow)
```

Comment:
...didn't I see this before...? ;-)

Reply:
I needed the changes from a previous PR in order to continue.